presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -479,9 +479,12 @@ else if (!partitionColumnNames.contains(name)) {
         table.setPartitionKeys(partitionColumns);
         table.setSd(sd);
 
-        PrivilegeGrantInfo allPrivileges = new PrivilegeGrantInfo("all", 0, tableOwner, PrincipalType.USER, true);
+        PrivilegeGrantInfo selectPrivilege = new PrivilegeGrantInfo("select", 0, tableOwner, PrincipalType.USER, true);
```

Comment:
It could be after the Presto auth is implemented, the docs says "ALL PRIVILEGES – gives all privileges (gets translated into all the above privileges)" - it means that literally, when creating a table in Hive, it gets the four separate priviledges(S/I/U/D). Table created in Presto gets one 'ALL' privilege which does not exist anymore, and the table can not be read by Hive(throws exception about ALL not found or exist).